targets/TARGET_TOSHIBA/mbed_rtx.h

```diff
@@ -0,0 +1,32 @@
+/* mbed Microcontroller Library
+ * (C)Copyright TOSHIBA ELECTRONIC DEVICES & STORAGE CORPORATION 2017 All rights reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MBED_MBED_RTX_H
+#define MBED_MBED_RTX_H
+
+#if defined(TARGET_TMPM066)
+
+#ifndef INITIAL_SP
+#define INITIAL_SP                        (0x20004000UL)
+#endif
+
+#ifdef TOOLCHAIN_GCC_ARM
```

Comment:
GCC tool chain usually generates bigger code than other tool chains. Because of this few of the test cases need more memory on the Heap. That's why, we have reduced mbed main stack size to 2KB from 4KB for our target.